crates/zksync_os_snark_prover/src/main.rs

```diff
@@ -39,8 +40,10 @@ enum Commands {
     },
 
     RunProver {
-        #[arg(short, long)]
-        sequencer_url: Option<String>,
+        /// List of sequencer URLs to poll for tasks (e.g., "http://<IP>:<PORT>")
+        /// The prover will poll sequencers in round-robin fashion
+        #[arg(short, long, alias = "sequencer-url", value_delimiter = ',', default_value = "http://localhost:3124", value_parser = clap::value_parser!(Url))]
+        sequencer_urls: Vec<Url>,
```

Comment:
As mentioned before, I'd like this to stay backwards compatible. I believe this is the case with `alias`, but please confirm.
The ask is -- whomever wants to upgrade, may require to change this as well (also, would be needed in integration tests, if we used this binary)